doc/articles/getting-started/counterapp/get-started-counter-csharp-mvux.md

```diff
@@ -4,7 +4,7 @@ uid: Uno.Workshop.Counter.CSharp.MVUX
 
 # Counter App using C# Markup and MVUX
 
-[Download the complete C# Markup + MVUX sample](https://github.com/unoplatform/Uno.GettingStartedTutorial/tree/master/src/Counter/CSharp-MVUX)
+[Download the complete C# Markup + MVUX sample](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter/CSharp-MVUX)
```

Comment:
Should we instead use a permalink? It will generally be better so that people that bookmarks the link can still find it easily in future.

cc @jeromelaban @agneszitte

Reply:
I think it's fine for now, we'll always point to the latest of our content.